src/main/java/com/commercetools/sync/services/CategoryService.java

```diff
@@ -24,7 +24,7 @@
      *         category keys -&gt; ids
      */
     @Nonnull
-    CompletionStage<Map<String, String>> cacheKeysToIds();
+    CompletionStage<Set<String>> loadExistingCategoryKeys();
```

Comment:
I would not change this method's name and structure as this method will be changed later and we need to fetch only keys from the batch for better performance. If you need a set maybe change it in the CategorySync.

Reply:
85e84de1